### PR TITLE
fix TLS_CACERTDIR on ldap.conf to take the pillar-submitted value

### PR DESCRIPTION
**Pull request by tampakrap**

### CODE REVIEW

**Comment by 0xf10e** [comment]
:+1:

File: openldap/files/ldap.conf [hunk 1]
```diff
@@ -21,12 +21,11 @@ URI    {{ salt['pillar.get']('openldap:uri') }}
 #TIMELIMIT	15
 #DEREF		never
 {% if not salt['pillar.get']('openldap:tls_cacertdir', False) %}
-{#- only needed on Debian? #}
 # TLS certificates (needed for GnuTLS)
 TLS_CACERT   {{ salt['pillar.get']('openldap:tls_cacert',
                     '/etc/ssl/certs/ca-certificates.crt') }}
 {%- else %}
-TLS_CACERTDIR /etc/openldap/certs
+TLS_CACERTDIR {{ salt['pillar.get']('openldap:tls_cacertdir') }}
```
Comment (aboe76) [inline openldap/files/ldap.conf]: shouldn't there be a default? 
like this:
{{ salt['pillar.get']('openldap:tls_cacertdir', '/etc/openldap/certs') }}

Comment (aboe76) [inline openldap/files/ldap.conf]: shouldn't there be a default? 
like this:
{{ salt['pillar.get']('openldap:tls_cacertdir', '/etc/openldap/certs') }}

Comment (tampakrap) [reply openldap/files/ldap.conf]: fixed

**Comment by aboe76** [comment]
@tampakrap thanks for this

**Pull request merged**